### ISSUE TITLE
Properties with parameters support

### ISSUE THREAD

**Issue by greenozon** [icsharpcode/ILSpy#2003]
I've got a question - does ILSpy support properties with parameters?

https://visualstudiomagazine.com/blogs/tool-tracker/2015/10/properties-with-parameters.aspx

I'm asking cause looks like there is an issue, eg:
![image](https://user-images.githubusercontent.com/765343/81493648-dee70880-92aa-11ea-92dd-2b86f5e5c752.png)

at the same time IL shows that there are input args!

![image](https://user-images.githubusercontent.com/765343/81493670-050ca880-92ab-11ea-9a77-3a370bc9e275.png)

**Comment by siegfriedpammer** [comment]
See also #1089

The problem is that C# does not support indexed properties with names other than `Item`.

**Comment by greenozon** [comment]
Here is how it looks like in VB.net language

![image](https://user-images.githubusercontent.com/765343/81579435-d7138b00-93b4-11ea-939d-01a644743d44.png)


to summarize - is it VB.net only feature?  (properties with parameters)
if yes - feel free to close this item

(source from dnSpy that still has VB as supported language)

**Comment by siegfriedpammer** [comment]
>  is it VB.net only feature?

Yes, exactly. We will have to find a representation that fits with C# eventually. I will keep this issue open until then.

**Comment by greenozon** [comment]
Thanks!
maybe (as an idea) it 'll be good to print a comment before such a VB only construction if used in C# decompilation...
